src/containers/Application.js

```diff
@@ -22,10 +22,11 @@ import ConfigService from '../config/ConfigService';
 import HeaderService from '../services/HeaderService';
 import SidebarService from './Sidebar/service';
 import ThemeService from '../theme/ThemeService';
+import HelpService from "../services/HelpService";
 
 import AccessDeniedCard from '../modules/tenant/access/AccessDeniedCard';
 
-import { ADD_ALERT, SET_ADVANCED_MODE, CHANGE_HELP_URL } from '../actions';
+import {ADD_ALERT, SET_ADVANCED_MODE, HELP_DESCRIPTION} from '../actions';
```

Comment:
@Aringoaway Reconsider, if its not better to rename `description` to `content`, since in the future there will be also possible to specify a url, not just a "description"..

So here the action will be named as `HELP_CONTENT`